### PR TITLE
update `Keyboard.getLayoutMap()` exception section

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

`Keyboard.getLayoutMap()` may throw SecurityError in policy control check; see https://wicg.github.io/keyboard-map/#h-keyboard-getlayoutmap step2

### Additional details

On https://developer.mozilla.org/en-US/docs/Web/API/Keyboard/getLayoutMap
Related to https://wicg.github.io/keyboard-map/

### CODE REVIEW

File: files/en-us/web/api/keyboard/getlayoutmap/index.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ None.
 A {{jsxref('Promise')}} that resolves with an instance of
 {{domxref('KeyboardLayoutMap')}}.
 
+### Exceptions
+
+- `SecurityError` {{domxref("DOMException")}}
+  - : Thrown if the call is blocked by [Permission Policy](/en-US/docs/Web/HTTP/Permissions_Policy).
```
Comment (teoli2003) [inline files/en-us/web/api/keyboard/getlayoutmap/index.md:36]:
```suggestion
  - : Thrown if the call is blocked by a [permission policy](/en-US/docs/Web/HTTP/Permissions_Policy).
```

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by ayissijoachim122** [comment]
our role at ecwid.com  :  Consultant  ANALYSE  MARKETS
- What languages do you code in?ENGLISH  FRENCH   GERMANY
- How are you currently looking for security vulnerabilities?  l work  and SAST  If you are using a SAST tool why is it not working out as desired?
- What version control systems are you using?  SAST  is good  very very good please in any other plan
- Do you prefer the solution to be on-premise or in the cloud?  yes  l very si in cloud  any  global programme
- How many developers and security professionals do you have in your environment?   15    developers and security professionals
- Do you have a timeline in mind for the evaluation?  yes


===========================
Company: Fiverr International Ltd.
Ticker: FVRR
Exchange: NYSE
Last Trade: $21.11
Change: -0.13
% Change: -0.61%
Volume: 630,618
Day's High: $21.73
Day's Low: $21.04
Previous Close: $21.24
Investor Relations | Fiverr International Ltd.            
bloombergbusiness.com
https://investors.fiverr.com
Investor Relations | Fiverr International Ltd.
https://moodys.com
Moodys: NYSE:MCO
Last Price 312.88 Change 4.88
Volume 1,000,040 % Change 1.58%
Day High 314.98 52 Week High 363.19
Day Low 307.23 52 Week Low 251.98
Today's Open 308.69 Previous Close 308.00
11/1/2023 4:00 PM
The Investor Relations website contains information about Fiverr International Ltd.'s business for stockholders,...

  email  ayissijoachim.72@gmail.com


https://ayissijoachim.postman.co/manage-public
E-mail
mobile@ecwid.com
place
Adresse
https://www.societe.com
verified_user
Règles de confidentialité
https://www.lightspeedhq.com/legal/privacy-policy/
https://share.vimeo.com/ayissijoachim.72
Dear Manager,

 thank you for visiting and validating my company from my city and my account has been checked all correct thank you thank you but I would have to send you other documents such as an administrative letter from our company with the members of administration and company stamp making liver thank you I beg you to be able to program all my funds for bank investment real estate investment bank loan lottery in bank States A States for program and investment loan and repayable to the Company Establishment Ayissi Joachim cella will be for life and also for the other investment in the program with my country Cameroon cella guide has worked with reliable investment agreements such as agriculture tourism trade and transport a general guide with global business climates with the world here is what we intend to do


Contact: AYISSI JOACHIM
E-mail: xxxjoachim122@yahoo.com
CellPhone: 237694616643
Software: Rent - Payment Platform Whitelabel
Country: Cameroon - Cameroon e Cameroun
Message: Name: AYISSI Joachim
              Address e_mail : ayissi_joachim@yahoo.com
              Phone. : (237) 677 89 16 02/694 61 66 43


Bank  Name : Union  Bank  PLC
BANK  ADRESSE: PObox 15569  Douala  Cameoon
Bank Account Name :   AYISSI  JOACHIM
Bank ACCOUNT  NUMBER  : 00410017574
BANK SWIFT : UCMACMCX
IBAN :CM21  10023  00040  00410017574  54

____________________________________________________
NEW IDENTIFICATION
              Name: ETABISSEMENT  AYISSI  JOACHIM
              Address e_mail : xxxjoachim122@yahoo.com
              BP. : 17474  DOUALA - Cameroon
              Phone. : (237) 677 89 16 02
              Bank Name - UNITED  BANK OF AFRICA
               IBAN :  CM21 10033  05207  07016001289  65
            AccounT ,Naime ; ETS  AYISSI  JOACHIM
            Account  Number ; 07016001289
            BANK CODE      SWIFT : UNAFCMCX
_________________________________________________

Bank  Name : Union  Bank  PLC
BANK  ADRESSE: PObox 15569  Douala  Cameoon
Bank Account Name :   AYISSI  JOACHIM
Bank ACCOUNT  NUMBER  : 00410017574
BANK SWIFT : UCMACMCX
IBAN :CM21  10023  00040  00410017574  54

Logo Google Cloud Bienvenue
Vous travaillez dans
.@JoachimAyi47226
https://www.instagram.com/ayissi_joachim
https://twitter.com/JoachimAyi47226
https://www.facebook.com/joachim.ayissi
https://www.pinterest.com/ayissijoachim/
https://www.linkedin.com/ayissijoachim
https://www.youtube.com/@ayissijoachim4632
https://github.com/ayissijoachim122
  ayissijoachim122/AYISSI-JOACHIM
Numéro du projet : 868342099878
ID du projet : confident-abode-401217
Tableau de bord
Recommandations
@ayissijoachim sur #Vimeo https://vimeo.com/876431318

- Identification requirements in the provisions of the new Regulations
in force in CEMAC States, following IMF reforms.
- Provisions for the fight against money laundering and the financing
of terrorism ; justification of transfers above 10,000 (ten thousand)
USD more favorable for anenterprise than a physical person .
- Need to provide the KYC of the Principal (YOU) and th

**Comment by skyclouds2001** [comment]
> our role at ecwid.com : Consultant ANALYSE MARKETS
> 
> * What languages do you code in?ENGLISH  FRENCH   GERMANY
> * How are you currently looking for security vulnerabilities?  l work  and SAST  If you are using a SAST tool why is it not working out as desired?
> * What version control systems are you using?  SAST  is good  very very good please in any other plan
> * Do you prefer the solution to be on-premise or in the cloud?  yes  l very si in cloud  any  global programme
> * How many developers and security professionals do you have in your environment?   15    developers and security professionals
> * Do you have a timeline in mind for the evaluation?  yes
> 
> =========================== Company: Fiverr International Ltd. Ticker: FVRR Exchange: NYSE Last Trade: $21.11 Change: -0.13 % Change: -0.61% Volume: 630,618 Day's High: $21.73 Day's Low: $21.04 Previous Close: $21.24 Investor Relations | Fiverr International Ltd. bloombergbusiness.com https://investors.fiverr.com Investor Relations | Fiverr International Ltd. https://moodys.com Moodys: NYSE:MCO Last Price 312.88 Change 4.88 Volume 1,000,040 % Change 1.58% Day High 314.98 52 Week High 363.19 Day Low 307.23 52 Week Low 251.98 Today's Open 308.69 Previous Close 308.00 11/1/2023 4:00 PM The Investor Relations website contains information about Fiverr International Ltd.'s business for stockholders,...
> 
> email [ayissijoachim.72@gmail.com](mailto:ayissijoachim.72@gmail.com)
> 
> https://ayissijoachim.postman.co/manage-public E-mail [mobile@ecwid.com](mailto:mobile@ecwid.com) place Adresse https://www.societe.com verified_user Règles de confidentialité https://www.lightspeedhq.com/legal/privacy-policy/ https://share.vimeo.com/ayissijoachim.72 Dear Manager,
> 
> thank you for visiting and validating my company from my city and my account has been checked all correct thank you thank you but I would have to send you other documents such as an administrative letter from our company with the members of administration and company stamp making liver thank you I beg you to be able to program all my funds for bank investment real estate investment bank loan lottery in bank States A States for program and investment loan and repayable to the Company Establishment Ayissi Joachim cella will be for life and also for the other investment in the program with my country Cameroon cella guide has worked with reliable investment agreements such as agriculture tourism trade and transport a general guide with global business climates with the world here is what we intend to do
> 
> Contact: AYISSI JOACHIM E-mail: [xxxjoachim122@yahoo.com](mailto:xxxjoachim122@yahoo.com) CellPhone: 237694616643 Software: Rent - Payment Platform Whitelabel Country: Cameroon - Cameroon e Cameroun Message: Name: AYISSI Joachim Address e_mail : [ayissi_joachim@yahoo.com](mailto:ayissi_joachim@yahoo.com) Phone. : (237) 677 89 16 02/694 61 66 43
> 
> Bank Name : Union Bank PLC BANK ADRESSE: PObox 15569 Douala Cameoon Bank Account Name : AYISSI JOACHIM Bank ACCOUNT NUMBER : 00410017574 BANK SWIFT : UCMACMCX IBAN :CM21 10023 00040 00410017574 54
> 
> NEW IDENTIFICATION Name: ETABISSEMENT AYISSI JOACHIM Address e_mail : [xxxjoachim122@yahoo.com](mailto:xxxjoachim122@yahoo.com) BP. : 17474 DOUALA - Cameroon Phone. : (237) 677 89 16 02 Bank Name - UNITED BANK OF AFRICA IBAN : CM21 10033 05207 07016001289 65 AccounT ,Naime ; ETS AYISSI JOACHIM Account Number ; 07016001289 BANK CODE SWIFT : UNAFCMCX
> 
> Bank Name : Union Bank PLC BANK ADRESSE: PObox 15569 Douala Cameoon Bank Account Name : AYISSI JOACHIM Bank ACCOUNT NUMBER : 00410017574 BANK SWIFT : UCMACMCX IBAN :CM21 10023 00040 00410017574 54
> 
> Logo Google Cloud Bienvenue Vous travaillez dans .@JoachimAyi47226 https://www.instagram.com/ayissi_joachim https://twitter.com/JoachimAyi47226 https://www.facebook.com/joachim.ayissi https://www.pinterest.com/ayissijoachim/ https://www.linkedin.com/ayissijoachim https://www.youtube.com/@ayissijoachim4632 https://github.com/ayissijoachim122 ayissijoachim122/AYISSI-JOACHIM Numéro du projet : 868342099878 ID du projet : confident-abode-401217 Tableau de bord Recommandations @ayissijoachim sur #Vimeo https://vimeo.com/876431318
> 
> * Identification requirements in the provisions of the new Regulations
>   in force in CEMAC States, following IMF reforms.
> * Provisions for the fight against money laundering and the financing
>   of terrorism ; justification of transfers above 10,000 (ten thousand)
>   USD more favorable for anenterprise than a physical person .
> * Need to provide the KYC of the Principal (YOU) and th

???

**Pull request merged**